README.md

```diff
@@ -19,23 +19,22 @@ A Fast and Flexible Static Site Generator built with love by [spf13](http://spf1
 ## Overview
 
 Hugo is a static HTML and CSS website generator written in [Go][].
-It is optimized for speed, easy use and configurability.
+It is optimized for speed, ease of use, and configurability.
 Hugo takes a directory with content and templates and renders them into a full HTML website.
 
-Hugo relies on Markdown files with front matter for meta data.
-And you can run Hugo from any directory.
+Hugo relies on Markdown files with front matter for metadata, and you can run Hugo from any directory.
 This works well for shared hosts and other systems where you don’t have a privileged account.
 
 Hugo renders a typical website of moderate size in a fraction of a second.
 A good rule of thumb is that each piece of content renders in around 1 millisecond.
 
-Hugo is designed to work well for any kind of website including blogs, tumbles and docs.
+Hugo is designed to work well for any kind of website including blogs, tumbles, and docs.
 
 #### Supported Architectures
 
-Currently, we provide pre-built Hugo binaries for Windows, Linux, FreeBSD, NetBSD and macOS (Darwin) and [Android](https://gist.github.com/bep/a0d8a26cf6b4f8bc992729b8e50b480b) for x64, i386 and ARM architectures.
+Currently, we provide pre-built Hugo binaries for Windows, Linux, FreeBSD, NetBSD, macOS (Darwin), and [Android](https://gist.github.com/bep/a0d8a26cf6b4f8bc992729b8e50b480b) for x64, i386 and ARM architectures.
 
-Hugo may also be compiled from source wherever the Go compiler tool chain can run, e.g. for other operating systems including DragonFly BSD, OpenBSD, Plan&nbsp;9 and Solaris.
```

Comment:
I believe the `&nbsp;` is intentional and makes sense so the word never breaks across multiple lines. With `&nbsp;` "Plan 9" is always on the same line.

Reply:
My apologies; not seeing it anywhere else, I had presumed it was a formatting issue committed from an unusual editor!